recipes-external/external-toolchain/external-toolchain.bb

```diff
@@ -10,6 +10,8 @@ LICENSE = "CLOSED"
 PN .= "-${TRANSLATED_TARGET_ARCH}"
 SKIPPED = "1"
 SKIPPED:tcmode-external = "0"
+INHIBIT_PACKAGE_STRIP = "1"
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
```

Comment:
After this we can probably drop the INSANE_SKIP for already-stripped in the same file below?

Reply:
You are talking about this https://github.com/MentorEmbedded/meta-sourcery/blob/2eea17a68d4067612c22aae03322086de28616ce/recipes-external/external-toolchain/external-toolchain.bb#L32

Reply:
@Noor-Ahsan yep

Reply:
If we also inhibit the sys root strip, then yes, otherwise it'll still warn in do_populate_sysroot. updated. Thanks